packages/itmat-ui-react/src/components/lxd/lxd.instance.list.tsx

```diff
@@ -0,0 +1,382 @@
+        onError: (error) => {
+            message.error(`Failed to update instance configuration: ${error.message}`);
+        }
+    });
+
+    const [systemStats, setSystemStats] = useState({
+        cpu: '',
+        memory: '',
+        storage: '',
+        gpu: '',
+        network: '',
+        pci: ''
+    });
+
+    const openUpdateConfigModal = (instance: LXDInstanceType) => {
+        setEditingInstance(instance);
+        setIsUpdateConfigModalOpen(true);
+    };
+
+    // Function to show the modal with the instance details
+    const showInstanceDetails = (instance: LXDInstanceType) => {
+        setSelectedInstance(instance);
+        setIsInstanceDetailsModalVisible(true);
+    };
+
+
+    useEffect(() => {
+        axios.get('/lxd')
+            .then(async response => {
+                const instances = response.data.data;
+
+                setInstances(instances.map((instance) => ({
+                    ...instance,
+                    key: instance.name
+                })));
+                setLoading(false);
+            })
+            .catch(error => {
+                const messageText = error instanceof Error ? error.message : 'An unknown error occurred';
+                message.error('Failed to load instance state: ' + messageText);
+                setLoading(false);
+            });
+    }, []);
+
+    useEffect(() => {
+        axios.get('/lxd/resources')
+            .then(response => {
+                const { data } = response.data; // Adjust according to the actual response structure
+                setSystemStats({
+                    cpu: data.cpu,
+                    memory: data.memory,
+                    storage: data.storage,
+                    gpu: data.gpu,
+                    network: data.network,
+                    pci: data.pci
+                });
+            })
+            .catch(error => {
+                console.error('Error fetching system information:', error);
+            });
+    }, []);
+
+
+    if (creatingInstance) {
+        // Pass a callback function to toggle back to the list view
+        return <CreateInstance onInstanceCreated={() => setCreatingInstance(false)} />;
+    }
+
+    // Function to start or stop an instance
+    const startOrStopInstance = async (instanceName, action) => {
+        try {
+            await axios.put(`/lxd/instances/${instanceName}/action`, {
+                action: action // 'start' or 'stop'
+            });
+            message.success(`Instance ${instanceName} ${action}ed`);
+            refreshInstancesList(); // Refresh the instances list after the action
+        } catch (error) {
+            const messageText = error instanceof Error ? error.message : 'An unknown error occurred';
+            message.error(`Failed to ${action} instance ${instanceName}: ${messageText}`);
+        }
+    };
+
+    const deleteInstance = async (instanceName) => {
+        try {
+            await axios.delete(`/lxd/instances/${instanceName}`);
```

Comment:
This probably won't yield the correct URL destination based on the new domain work being done